src/ToDo/Business/Entities/UserContext.cs

```diff
@@ -0,0 +1,9 @@
+﻿
+namespace ToDo.Business.Entities
+{
+	public record UserContext
+	{
+		public string? name { get; set; }
```

Comment:
Changes:
- Use correct casing for properties
- Mark properties as init so they function correctly as record 
eg public string? Name {get; init;}